ops/terraform/services/pipeline/main.tf

```diff
@@ -209,3 +209,7 @@ module "bfd_pipeline_slis" {
   aws_kms_key_id  = local.kms_key_id
   etl_bucket_id   = aws_s3_bucket.this.id
 }
+
+module "bfd_pipeline_dashboard" {
+  source = "./modules/bfd_pipeline_dashboard"
+}
```

Comment:
Having worked with a handful of ephemeral environments recently, I think toggling the inclusion could be helpful for this and other child modules for non path-to-production environments; it ought to be included by default in `test`, `prod-sbx`, and `prod`, but excluded by default elsewhere.

Reply:
Agreed! I'll follow the example for ETL user creation and add a `pipeline` module variable for forcing dashboard creation in non-established environments. I'll update this such that the module is only included if we're in an established environment _or_ the aforementioned override/force variable is `true`.